api/src/LinkyLink/Startup.cs

```diff
@@ -12,6 +12,8 @@ namespace LinkyLink
 {
     public class Startup
     {
+        readonly string _CORSPolicyName = "OHCORSPolicyName";
```

Comment:
Standard c# naming conventions for private members should follow camel casing rules.  `corsPolicyName`.  This looks more like a constant in which i would then follow pascal casing like `CorsPolicyName`

`private const string CorsPolicyName = "OHCORSPolicyName"`

Reply:
Yea, it is a constant. I did this to follow the same naming convention used in the backend out of consistency.  Would you still recommend changing it?

Reply:
I will merge the PR to unblock My and can create a separate PR if we need to update the naming convention across the backend code base